vendor/bat-native-ads/src/bat/ads/internal/purchase_intent/funnel_sites.h

```diff
@@ -0,0 +1,1820 @@
+// retrieve via getter
+static const std::vector<std::string>& _funnel_site_segments = {
+  "automotive purchase intent by category-compact pickup",
+  "automotive purchase intent by category-compact suv",
+  "automotive purchase intent by category-compact van",
+  "automotive purchase intent by category-entry compact car",
+  "automotive purchase intent by category-entry luxury car",
+  "automotive purchase intent by category-entry midsize car",
+  "automotive purchase intent by category-entry sports car",
+  "automotive purchase intent by category-fullsize suv",
+  "automotive purchase intent by category-hd fullsize pickup",
+  "automotive purchase intent by category-ld fullsize pickup",
+  "automotive purchase intent by category-luxury sports car",
+  "automotive purchase intent by category-luxury suv",
+  "automotive purchase intent by category-mid luxury car",
+  "automotive purchase intent by category-midsize suv",
+  "automotive purchase intent by category-premium compact car",
+  "automotive purchase intent by category-premium luxury car",
+  "automotive purchase intent by category-premium midsize car",
+  "automotive purchase intent by category-premium sports car",
+  "automotive purchase intent by make-acura",
+  "automotive purchase intent by make-audi",
+  "automotive purchase intent by make-bmw",
+  "automotive purchase intent by make-buick",
+  "automotive purchase intent by make-cadillac",
+  "automotive purchase intent by make-chevrolet",
+  "automotive purchase intent by make-chrysler",
+  "automotive purchase intent by make-dodge",
+  "automotive purchase intent by make-fiat",
+  "automotive purchase intent by make-ford",
+  "automotive purchase intent by make-gmc",
+  "automotive purchase intent by make-honda",
+  "automotive purchase intent by make-hyundai",
+  "automotive purchase intent by make-infiniti",
+  "automotive purchase intent by make-jaguar",
+  "automotive purchase intent by make-jeep",
+  "automotive purchase intent by make-kia",
+  "automotive purchase intent by make-land rover",
+  "automotive purchase intent by make-lexus",
+  "automotive purchase intent by make-lincoln",
+  "automotive purchase intent by make-mazda",
+  "automotive purchase intent by make-mercedes-benz",
+  "automotive purchase intent by make-mini",
+  "automotive purchase intent by make-mitsubishi",
+  "automotive purchase intent by make-nissan",
+  "automotive purchase intent by make-porsche",
+  "automotive purchase intent by make-ram",
+  "automotive purchase intent by make-saab",
+  "automotive purchase intent by make-scion",
+  "automotive purchase intent by make-smart",
+  "automotive purchase intent by make-subaru",
+  "automotive purchase intent by make-suzuki",
+  "automotive purchase intent by make-toyota",
+  "automotive purchase intent by make-volkswagen",
+  "automotive purchase intent by make-volvo"
+};
+
+static const std::vector<FunnelSiteInfo> _automotive_funnel_sites = {
+  FunnelSiteInfo(
```

Comment:
Don't need to call constructor explicitly. Use std::initializer_list instead.

Reply:
@gdregalo like this? `FunnelSiteInfo {_funnel_site_segments,  "http://autotrader.com",  1}`

What's the benefit here? These static lists will be pulled out and refactored with the next piece of work to support "out of band" updates